.github/workflows/release.yaml

```diff
@@ -60,4 +60,4 @@ jobs:
       - name: Setup gcloud
         uses: google-github-actions/setup-gcloud@e30db14379863a8c79331b04a9969f4c1e225e0b
       - name: Upload To Release Bucket
-        run: gsutil rsync -r modules/sync gs://buf-modules
+        run: gsutil -m rsync -c -r modules/sync gs://buf-modules
```

Comment:
Global options (`-m`): https://cloud.google.com/storage/docs/gsutil/addlhelp/GlobalCommandLineOptions#options
Rsync options (`-c`): https://cloud.google.com/storage/docs/gsutil/commands/rsync#options

I really think it would be safe if we ran with `-i` which ignores mtime entirely and just skips any files which exist at the destination already, however we can try starting with `-c` to see what the impact on release time is. Right now I've confirmed that every time it copies all of the files.

Reply:
Actually `-i` won't work for files like `state.json` that change with each update. It would only work for the CAS files.